app/Entities/Post.php

```diff
@@ -34,9 +34,7 @@ public function link()
 
     public function setReplyTo(?string $value = null)
     {
-        if (empty($value)) {
-            $this->attributes['reply_to'] = null;
-        }
+        $this->attributes['reply_to'] = $value ?: null;
```

Comment:
```suggestion
        $this->attributes['reply_to'] ??= $value;
```

Reply:
The goal here is to convert an empty string to `null`.